src/content-helper/editor-sidebar/smart-linking/review-modal/component-block-preview.tsx

```diff
@@ -0,0 +1,203 @@
+/**
```

Comment:
If I understand correctly, this file is mostly copy/paste from `src/content-helper/editor-sidebar/smart-linking/review-modal/component-suggestion.tsx` without changes?

Reply:
Yeah, pretty much. I have extracted this to its own file so it can be easily reused in both outbound and inbound dialogs. The only significant change is the new `useOriginalBlock` parameter, which if it's set to true, will use the original block for the preview instead of cloning it. This is very useful for the inbound smart links, since the paragraph block that is previewed does not exist in the Block Editor context, and therefore the clone is not needed.

Reply:
Thanks for explaining!